cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -345,7 +346,31 @@ resource {{bicepName $name}}OperatorRoleAssignment 'Microsoft.Authorization/role
     roleDefinitionId:  subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '230815da-be43-4aae-9cb4-875f7bd000aa')
   }
 }
-{{- end}}
+{{range $cname := $value.Databases}}
+resource {{bicepName $cname}} 'Microsoft.DocumentDB/databaseAccounts/sqlDatabases@2023-04-15' = {
+  name: '{{$cname}}'
+  location: location
+  tags: union(tags, {'aspire-resource-name': '{{$cname}}'})
+  parent: {{bicepName $name}}
+  properties: {
+    resource: {
+      id: '{{$cname}}'
+    }
+  }
+}
+
+// SQL DB Contributor Role
```

Comment:
Do we still need this role assignment given that we are not on an RBAC plan now?

Reply:
we don't
I keep it b/c we might want to move back to it in the future. 
Would you prefer removing it for now?

Reply:
> Would you prefer removing it for now?

Yeah, let's remove it until we are actually fully on the RBAC plan...